v4/integrations/nrgrpc/nrgrpc_server_test.go

```diff
@@ -108,7 +108,7 @@ func TestUnaryServerInterceptor(t *testing.T) {
 		},
 		{
 			Name:     "TestApplication/DoUnaryUnary",
-			ParentID: internal.MatchNoParent,
+			ParentID: internal.MatchAnyParent,
```

Comment:
#### Bug 2

When a distributed tracing payload is successfully received when calling `txn.AcceptDistributedTraceHeaders`, we created a `TransportDuration` metric. Therefore, the presence of this metric would indicate that _every_ span in the transaction should have a parent.

This lead me to uncover that we should be calling `AcceptDistributedTraceHeaders` in `SetWebRequest`. We haven't technically implemented that method, but I did add the call to Accept in it in order to improve our testing.